contracts/scripts/operational/setRemoteTokenBridgeTask.ts

```diff
@@ -11,24 +11,22 @@ import { getDeployedContractOnNetwork } from "../../common/helpers/readAddress";
     *******************************************************************************************
     SEPOLIA_PRIVATE_KEY=<key> \
     INFURA_API_KEY=<key> \
-    npx hardhat transferOwnershipAndSetRemoteTokenBridge \
+    npx hardhat setRemoteTokenBridge \
     --safe-address <address> \
     --remote-token-bridge-address <address> \
     --token-bridge-address <address> \
-    --token-bridge-proxy-admin-address <address> \
     --remote-network sepolia \
     --network linea_sepolia
     *******************************************************************************************
 */
 
 task(
-  "transferOwnershipAndSetRemoteTokenBridge",
+  "setRemoteTokenBridge",
   "Transfers the ownership of TokenBridge and Proxy Admin to the Safe address and also sets the remoteTokenBridge address.",
 )
   .addParam("safeAddress")
```

Comment:
Should we remove this param, given it's not used anymore?

Reply:
done